Signal/src/ViewControllers/LoadingViewController.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+import Foundation
+import PromiseKit
+
+// The initial presentation is intended to be indistinguishable from the Launch Screen.
+// After a delay we present some "loading" UI so the user doesn't think the app is frozen.
+@objc
+public class LoadingViewController: UIViewController {
+
+    var logoView: UIImageView!
+    var topLabel: UILabel!
+    var bottomLabel: UILabel!
+
+    override public func loadView() {
+        self.view = UIView()
+        view.backgroundColor = UIColor.ows_materialBlue
+
+        self.logoView = UIImageView(image: #imageLiteral(resourceName: "logoSignal"))
+        view.addSubview(logoView)
+
+        logoView.autoCenterInSuperview()
```

Comment:
Now that we're not using the actual Launch Screen, we can more easily add behavior.

However, it's now possible for the loading screen to look distinguishable from the Launch screen, which would be undesirable.

But it's such a simple view and we see it so often that I'm not very worried about breaking it.